The channel used with signal.Notify should be buffered

The `staticcheck` static analysis tool noticed the following warning:

~~~
service.go:325:15:warning: the channel used with signal.Notify should be buffered (SA1017) (staticcheck)
~~~

Some background info is at https://golang.org/pkg/os/signal/#Notify .

It looks to me like syscall.SIGINT and syscall.SIGTERM only need to have a buffer size of 1, since upon receiving the first instance of either, the program will exit.  However, I'm not certain of this since I'm not very familiar with the code here.  The example code at https://gobyexample.com/signals uses a buffer size of 1.

--- a/service.go
+++ b/service.go
@@ -321,7 +321,7 @@ func (info *Info) runInteractively() error {
 		doneChan <- err
 	}()
 
-	sig := make(chan os.Signal)
+	sig := make(chan os.Signal, 1)
 	signal.Notify(sig, syscall.SIGINT, syscall.SIGTERM)
 
 	var exitErr error

